operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java

```diff
@@ -421,7 +451,7 @@ private static Uri uri(FHIROperationContext operationContext, String uriPath) {
                 .build();
     }
     
-    private void addIncludesSearchParameters(String compartmentMemberType, MultivaluedMap<String, String> searchParameters)
+    private void addIncludesSearchParameters(String compartmentMemberType, MultivaluedMap<String, String> searchParameters, List<String> extraResources)
     throws Exception {
```

Comment:
This is starting to feel like it should be a separate managed class. 
The class would have instance level parameters for managing this content as the decisions are made.

Reply:
Perhaps a refactor in the future?